### PR TITLE
Restructure

### PR DESCRIPTION
**Pull request by osi-amit**

### CODE REVIEW

File: PI-Vision-Extensibility/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+If your support question or issue is related to an OSIsoft product (error message, problem with product configuration, etc.), please open a case with OSIsoft Tech Support through the [OSIsoft Customer Portal](https://my.osisoft.com/).
+
+If your support question or issue is related to a non-modified sample (or test) or sample documentation; please email [Samples@osisoft.com](mailto:Samples@osisoft.com). 
+
+### License
+
+The PI Vision Extensibility Samples in this repository are licensed under the [Apache 2 license](https://github.com/osisoft/OSI-Samples/blob/master/LICENSE.md).
+
+[![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/PI%20Vision/extensibility-sample.unit-tests?branchName=47718_ExtensibilitySample)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=1311&branchName=47718_ExtensibilitySample)
```
Comment (derekendres) [inline PI-Vision-Extensibility/README.md]: This build status doesn't make sense I would either get rid of it for now, or setup a table for when there are more symbols

**Review by derekendres — COMMENTED** [review]
might be good to fix the typos on the base readme (and the links) its vison instead of vision

File: PI-Vision-Extensibility/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# PI Vision Extensibility
+This repository contains sample HTML/CSS/JavaScript for creating custom symbols for PI Vision.
+
+## Overview
+The sample code contained in this repository provides code to extend PI Vision with new custom symbols.
```
Comment (derekendres) [inline PI-Vision-Extensibility/README.md]: I think this line is off and I would say something like inside this repository are custom symbol samples and the PI Vision Extensiblity guide, which is available under <a href="docs">.docs</a>

File: PI-Vision-Extensibility/README.md (hunk 1 above)
Comment (osi-amit) [reply PI-Vision-Extensibility/README.md]: I'll remove the Build Status section for now.

File: PI-Vision-Extensibility/README.md (hunk 2 above)
Comment (osi-amit) [reply PI-Vision-Extensibility/README.md]: OK, will make this change

**Review by derekendres — APPROVED** [review]

**Pull request merged**